zuul-core/src/main/java/com/netflix/zuul/netty/server/ClientResponseWriter.java

```diff
@@ -247,7 +247,7 @@ else if (evt instanceof CompleteEvent) {
             }
             else {
                 if (isHandlingRequest) {
-                    LOG.warn("Received complete event while still handling the request. With reason: " + reason.name() + " -- " +
+                    LOG.info("Received complete event while still handling the request. With reason: " + reason.name() + " -- " +
```

Comment:
I think this is likely due to clients that employ http pipelining, which zuul doesn't support. #1045 should ensure that the `reason` in the message is populated accurately, when that is indeed the case.

That said, the downgrade LGTM.

Reply:
> That said, the downgrade LGTM. Probably consider DEBUG.

I just updated the code. It now uses LOG.debug